src/SpotifyApi.NetCore/FollowApi.cs

```diff
@@ -132,14 +128,193 @@ public FollowApi(HttpClient httpClient) : base(httpClient)
+        /// <param name="limit">Optional. The maximum number of items to return. Default: 20. Minimum: 1. Maximum: 50.</param>
+        /// <param name="after">Optional. The last artist ID retrieved from the previous request.</param>
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedArtists"/> object.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/get-followed/
+        /// </remarks>
+        public async Task<PagedArtists> GetUsersFollowedArtists(
+            int limit = 20,
+            string after = null,
+            string accessToken = null
+            ) => await GetUsersFollowedArtists<PagedArtists>(limit, after, accessToken);
+
+        /// <summary>
+        /// Get User's Followed Artists
+        /// </summary>
+        /// <param name="limit">Optional. The maximum number of items to return. Default: 20. Minimum: 1. Maximum: 50.</param>
+        /// <param name="after">Optional. The last artist ID retrieved from the previous request.</param>
+        /// <returns>A Task that, once successfully completed, returns a full <see cref="PagedArtists"/> object.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/get-followed/
+        /// </remarks>
+        public async Task<T> GetUsersFollowedArtists<T>(
+            int limit = 20,
+            string after = null,
+            string accessToken = null
+            )
+        {
+            if (limit < 1 || limit > 50) throw new
+                    ArgumentException("A minimum of 1 and a maximum of 50 artist ids can be sent.");
+
+            UriBuilder builder = new UriBuilder($"{BaseUrl}/me/following");
+            builder.AppendToQuery("type", "artist");
+            builder.AppendToQuery("limit", limit);
+            if (!string.IsNullOrWhiteSpace(after))
+            {
+                builder.AppendToQuery("after", after);
+            }
+            return await GetModel<T>(builder.Uri, accessToken);
+        }
+        #endregion
+
+        #region UnfollowArtistsOrUsers
+        /// <summary>
+        /// Unfollow Artists or Users
+        /// </summary>
+        /// <param name="artistIds">A comma-separated list of the artist Spotify IDs. A maximum of 50 artist IDs can be sent in one request. A minimum of 1 artist id is required.</param>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/follow/unfollow-artists-users/
+        /// </remarks>
+        public async Task UnfollowArtists(
+            string[] artistIds,
+            string accessToken = null
+            )
+        {
+            if (artistIds?.Length < 1 || artistIds?.Length > 50) throw new
+                    ArgumentException("A minimum of 1 and a maximum of 50 artist ids can be sent.");
+
+            UriBuilder builder = new UriBuilder($"{BaseUrl}/me/following");
```

Comment:
Use `var`

Reply:
Hi Daniel,

Will change this everywhere.  If I understand the logic you will use var when the words match meaning in this case since UriBuilder string matches == UriBuilder there is no point typing it again so use var. Is that it? Because technically in my code there is no statement which will not resolve to a type in assignment operations hence technically all should be var.

Thanks for your patience and time.

Best Regards,

Akshay Srinivasan

Reply:
That is correct, use `var` wherever the Type is obvious.